### DISCUSSION TITLE
更换vpn之后，无法登录。添加完API之后，登录后却无法导入音频文件。原来的学习记录全部消失了。

### DISCUSSION THREAD

**Discussion by thinkbankcoin** [ZuodaoTech/everyone-can-use-english#962] [General]
看截图提示，是不是除了添加API，还要添加代理呢？代理填写什么呢？
![Uploading 截屏2024-08-13 08.58.15.png…]()

**Comment by an-lee** [comment]
API 更换为 https://api.getenjoyapp.com 是为了免代理，改了可以试试把 VPN 关掉看。
如果 VPN 可以全局代理，就不需要设置代理，如果不能，可以参考你 VPN 的说明，添加代理地址。

**Reply by thinkbankcoin** [reply]
这个问题我已经解决了，对的，我现在仍然使用后面新换的VPN，可以正常使用，而且最近的学记记录已经恢复了。我想应该是VPN只要开全局，如果出现网络错误，无法登录，应该是服务反映慢，我们需要重新关机，或者多次退出重新登录，就可以了，我昨天试了几次不行，今天重新开机，重新登录了2次，就成功了。如果VPN不开全局，你就向VPN供应商询问代理链接即可。
（供大家参考）有时候Telegram也是如此。
谢谢李导儿！！！
